### DISCUSSION TITLE
Setting cache dir

### DISCUSSION THREAD

**Discussion by evwltrs** [kubilus1/autoortho#45] [General]
Hi :),

Love the tool. However I was wondering if it was possible to update the cache dir on Windows as changing it in the configurator currently does nothing.

Thanks

**Comment by kubilus1** [comment]
Looks like a bug!  You could manually change this in the config file '.autoortho' in your Users directory.  I'll look into why setting this in the UI isn't working.  Thanks!!

**Comment by NNowakowski** [comment]
It seems like the settings in .autoortho do not have any effect either.
Despite changing all directory settings and them being confirmed in the terminal the files are still written to the default folder in home.
![image](https://user-images.githubusercontent.com/16933892/221353899-66914eb6-2449-41d3-8d1a-e83427635242.png)

**Reply by NNowakowski** [reply]
On further inspection the line responsible for populating the download location from the settings seems to be commented out entirely:
https://github.com/kubilus1/autoortho/blob/main/autoortho/downloader.py#L421
